src/stateManager/StateManager.ts

```diff
@@ -99,8 +99,7 @@ class StateManager {
     private dispatcher = new Map([
         [BlockValidationAction.DISPUTE, this.dispute],
         [BlockValidationAction.BROADCAST, this.broadcast],
-        [BlockValidationAction.NOT_ENOUGH_TIME, this.notEnoughTime],
-        [BlockValidationAction.SUCCESS, this.success]
+        [BlockValidationAction.NOT_ENOUGH_TIME, this.notEnoughTime]
```

Comment:
@daiagi missing merge with `dispute` to account for new changes - what I never understood, why doesn't github display updated files in a PR when the destination branch is updated (e.g. today's PR got merged into `dispute` and it's not reflected here)